test/compiler/pass/_list_ops.ods

```diff
@@ -15,8 +15,5 @@ do print(len([]))
 do print(len(ints))
 
 /* list cons test */
-do z = cons(5, ints)
```

Comment:
Now you have to use `::` to prepend to a list. `::` corresponds to the builtin function `_cons`, which can't be overwritten or called by the user.